### ISSUE TITLE
 Grid renders an empty space between fixed columns on column resizing

### ISSUE THREAD

**Issue by jmarcelfpg** [DevExpress/devextreme-reactive#2046]
I tried in your forum https://www.devexpress.com/Support/Center/Question/Details/T744004/fixedcolums-resizingcolumns-bug

but It has 1 day and I don't get any response yet

- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...

- [x] React Grid
- [ ] React Chart
- [ ] React Scheduler

### Current Behaviour

The Grid have a bug when you combine a few fixed columns in a row and resizing.
if you try to resize a column between fixed ones they stuck their position some times and a hole appears between them

### Steps to Reproduce

here the reproducible code, and after running just try to resize a fixed column in of all the fixed columns.
[JavaScript]
```
import React, { Component, Fragment } from "react";
import { withStyles } from "@material-ui/core/styles";
import Paper from "@material-ui/core/Paper";
import "@devexpress/dx-react-grid";
import {
  Grid,
  Table,
  TableHeaderRow,
  TableColumnResizing,
  TableFixedColumns,
} from "@devexpress/dx-react-grid-material-ui";

import columns from "./columns";
import data from "./data";

const styles = theme => ({
  div: {
    overflowY: "auto",
    overflowX: "hidden",
  },
});

class DevExpressGrid extends Component {
  constructor(props) {
    super(props);

    this.state = {
      columns: columns,
      rows: data.map((row, id) => Object.assign(row, { id })),
      tableColumnExtensions: [
        { columnName: "id", width: 130 },
        { columnName: "name", width: 140 },
        { columnName: "city", width: 210 },
        { columnName: "car", width: 240 },
        { columnName: "cost", width: 180 },
        { columnName: "sex", width: 50 },
      ],
      defaultColumnWidths: [
        { columnName: "id", width: 130 },
        { columnName: "name", width: 140 },
        { columnName: "city", width: 210 },
        { columnName: "car", width: 240 },
        { columnName: "cost", width: 180 },
        { columnName: "sex", width: 50 },
      ],
    };
  }

  render() {
    const {
      rows,
      columns,
      tableColumnExtensions,
      defaultColumnWidths,
    } = this.state;
    const { classes } = this.props;

    return (
      <Fragment>
        <div className={classes.div}>
          <Grid rows={rows} columns={columns}>
            <Table />
            <TableColumnResizing defaultColumnWidths={defaultColumnWidths} />
            <TableHeaderRow />
            <TableFixedColumns leftColumns={["id", "name", "sex", "city"]} />
          </Grid>
        </div>
      </Fragment>
    );
  }
}

const StyledDevExpressGrid = withStyles(styles)(DevExpressGrid);

export default () => <StyledDevExpressGrid />;
```

### Screenshots

![FixedColumns+ResizingColumns](https://user-images.githubusercontent.com/35382667/58345828-ece41500-7e1e-11e9-883c-f37524fffc3c.png)

### Environment

Create-react-app and I updated to latest react with yarn

    "@storybook/addon-actions": "^5.0.11",
    "@storybook/addon-knobs": "^5.0.11",
    "@storybook/addon-links": "^5.0.11",
    "@storybook/react": "^5.0.11"
    "@devexpress/dx-react-core": "^1.9.0",
    "@devexpress/dx-react-grid": "^1.9.0",
    "@devexpress/dx-react-grid-material-ui": "^1.9.0",
    "@material-ui/core": "^3.2.2",
    "@material-ui/icons": "^3.0.1",
    "react": "^16.6.0",

**Comment by churkin** [comment]
Hi @jmarcelfpg, 
I've fixed this problem, and you can check it with the next version.
Feel free to reopen this issue if the problem isn't resolved.

**Comment by jmarcelfpg** [comment]
@churkin the latest version before this commit was 1.11.0, and after the version still is this one.
your npm page https://www.npmjs.com/package/@devexpress/dx-core still show v1.11.0 and the date of the latest published version is before too.

I tried to use the command:
` yarn upgrade @devexpress/dx-react-core@latest @devexpress/dx-react-grid@latest @devexpress/dx-react-grid-material-ui@latest`

And It isn't changing to another version so the error is still there

**Comment by jmarcelfpg** [comment]
@churkin @NickMitrokhin 
Could you guy help me to reopen the issue?, it seems I don't have enough privileges to reopen it

Thanks

**Comment by jmarcelfpg** [comment]
Any update in this one?, @churkin , @MaximKudriavtsev , @NickMitrokhin

**Comment by MaximKudriavtsev** [comment]
@jmarcelfpg 

The next React Grid realize will be available on `npm` after one week.

**Comment by MaximKudriavtsev** [comment]
@jmarcelfpg,

We have released the [1.11.1 version](https://github.com/DevExpress/devextreme-reactive/blob/master/CHANGELOG.md#1111-2019-06-14) of DevExtreme Reactive Grid with this fix.
